featurebyte/service/feature_table_cache.py

```diff
@@ -443,11 +507,112 @@ async def create_or_update_feature_table_cache(
+            Graph definition
+        nodes: List[Node]
+            Input node names
+        output_view_details: TableDetails
+            Output table details
+        is_target : bool
+            Whether it is a target computation call
+        feature_list_id: Optional[PydanticObjectId]
+            Optional feature list id
+        serving_names_mapping: Optional[Dict[str, str]]
+            Optional serving names mapping if the observations set has different serving name columns
+            than those defined in Entities
+        progress_callback: Optional[Callable[[int, Optional[str]], Coroutine[Any, Any, None]]]
+            Optional progress callback function
+        """
+        await self.create_or_update_feature_table_cache(
+            feature_store=feature_store,
+            observation_table=observation_table,
+            graph=graph,
+            nodes=nodes,
+            is_target=is_target,
+            feature_list_id=feature_list_id,
+            serving_names_mapping=serving_names_mapping,
+            progress_callback=progress_callback,
+        )
+        cache_metadata = (
+            await self.feature_table_cache_metadata_service.get_or_create_feature_table_cache(
+                observation_table_id=observation_table.id,
+            )
+        )
+        cached_features = {
+            feature.definition_hash: feature.feature_name
+            for feature in cache_metadata.feature_definitions
+        }
+        hashes = await self.definition_hashes_to_nodes(graph, nodes)
+
+        db_session = await self.session_manager_service.get_feature_store_session(
+            feature_store=feature_store
+        )
+
+        request_column_names = sorted({col.name for col in observation_table.columns_info})
+        request_columns = [quoted_identifier(col) for col in request_column_names]
+        columns_expr = [
+            expressions.alias_(
+                quoted_identifier(cast(str, cached_features[definition_hash])),
+                alias=graph.get_node_output_column_name(node.name),
+                quoted=True,
+            )
+            for definition_hash, node in hashes.items()
+        ]
+        select_expr = (
+            expressions.select(quoted_identifier(InternalName.TABLE_ROW_INDEX))
+            .select(*request_columns)
+            .select(*columns_expr)
+            .from_(
+                get_fully_qualified_table_name(
+                    {
+                        "database_name": db_session.database_name,
+                        "schema_name": db_session.schema_name,
+                        "table_name": cache_metadata.table_name,
+                    }
+                )
+            )
+        )
+        create_view_exr = expressions.Create(
+            this=expressions.Table(
+                this=get_fully_qualified_table_name(output_view_details.dict()),
+            ),
+            kind="VIEW",
+            expression=select_expr,
+            replace=True,
```

Comment:
Is there a need to set replace? If not, probably safer to not set it to avoid overwriting unexpectedly.